app/views/schools/register_ect_wizard/change_email_address.html.erb

```diff
@@ -0,0 +1,2 @@
+<%= render 'email_address',
+            backlink_href: schools_register_ect_wizard_check_answers_path %>
```

Comment:
If you define `previous_step` in the step class `Schools::RegisterECTWizard::ChangeEmailAddressStep` then this will become `backlink_href: @wizard.previous_step_path` (exactly the same as `app/views/schools/register_ect_wizard/change_email_address.html.erb` so `backlink_href` can be moved inside the partial and the 2 view files just be:
`<%= render 'email_address' %>

Reply:
Initially, I was using `previous_step` methods, but I noticed that the register mentor files use path helpers instead. So, I updated it to follow the same approach for consistency.

Reply:
I'm going to create a PR to do it in the mentor journey. 
If you merge this one as it is I can address it as well as part of that PR.